test/checker.go

```diff
@@ -0,0 +1,56 @@
+package test
+
+import (
+	"fmt"
+	"reflect"
+
+	check "gopkg.in/check.v1"
+)
+
+// isInListChecker verifies if an element is present in the provided list.
+// The list must be a slice or array, and the element must be the same type as the list.
+//
+// For example:
+//
+//     c.Assert("b", IsInList, []string{"a", "b", "c"})
+//     c.Assert(2, IsInList, []int{1, 2, 3})
+//
```

Comment:
Suggest moving this doc comment to `var IsInList`. This aligns with the upstream implementations.
https://github.com/go-check/check/blob/v1/checkers.go#L221-L230

Reply:
Nice reminder, thanks!